src/app/components/game/interface/connected-players/connected-players.component.ts

```diff
@@ -33,7 +37,17 @@ export class ConnectedPlayersComponent implements OnInit, OnDestroy {
   // subscriptions
   playerIsDev$$: Subscription;
 
-  constructor(protected gameState: GameStateService, private userService: UserService, private router: Router) {}
+  // callbackIds
+  playerUpdateId: number;
```

Comment:
Ist playerUpdateId ein sinnvoller Name für die Referenznummer/Identifier eines MessageCallbacks auf einen REFRESH_COMMAND?

Reply:
You are right this was renamed to a designer naming scheme provided by an external free-lancer named Kevin